arches_lingo/media/js/views/root.js

```diff
@@ -10,7 +12,26 @@ const router = createRouter({
     routes,
 });
 
-createVueApplication(LingoApp).then(vueApp => {
+const LingoPreset = definePreset(ArchesPreset, {
+    components: {
+        button: {
+            root: {
+                label: {
+                    fontWeight: 600,
```

Comment:
I don't love the idea of overriding the font weight of all lingo buttons. Would it be a better idea to keep it standard and override on a case-by-case basis?

Reply:
Sure, let's try it.